modules/db/src/main/scala/gem/dao/StepDao.scala

```diff
@@ -398,6 +398,8 @@ object StepDao {
 
     object Gmos {
 
+      implicit val WavelengthMeta = WavelengthMetaAsNanometers
```

Comment:
So now there is no implicit `Meta[Wavelength]`, which may be annoying idk.  GMOS seems to like wavelengths expressed in nm while GNIRS uses μm.

Reply:
I think this is what we want.